packages/evm/src/precompiles/06-ecadd.ts

```diff
@@ -24,7 +30,8 @@ export function precompile06(opts: PrecompileInput): ExecResult {
     return OOGResult(opts.gasLimit)
   }
 
-  const returnData = unprefixedHexToBytes(ec_add(inputData))
+  const ret = ec_add(inputData)
+  const returnData = ret.slice(0, 2) === '0x' ? hexToBytes(ret) : unprefixedHexToBytes(ret)
```

Comment:
Note: (for all `ec*` files), it seems that if there is a return value, the string is not prefixed, but if there is no return value (error?) then 0x is returned. (So we need to do this check)